src/main.cpp

```diff
@@ -153,7 +154,40 @@ static std::optional<std::string> choose_socket(wl_display *display)
 
 static wf::config_backend_t *load_backend(const std::string& backend)
 {
-    auto [_, init_ptr] = wf::get_new_instance_handle(backend);
+    std::string config_plugin(backend);
+
+    if (backend.size())
```

Comment:
Is this not duplicating code from the plugin loader for normal plugins? Can we not refactor the code to avoid the duplication?

Reply:
Added two functions to `output/plugin-loader.[cpp|hpp ]` to avoid code duplication.